src/app/components/bigCard.tsx

```diff
@@ -0,0 +1,69 @@
+﻿import {Poppins} from 'next/font/google';
```

Comment:
General thoughts:

- Run prettier on your files so we can maintain consistency in codestyle (`yarn prettify`)
- For component based CSS, you can use the linaria CSS-in-JS framework we configured so that you don't need to use the inline style attribute. I've provided an example below for this